### PR TITLE
Add width & height to common declarations

### PR DESCRIPTION
**Pull request by lachlanjc**
Adds `minWidth`, `width`, & `height` to common declarations.

### CODE REVIEW

File: src/common-declarations.js [hunk 1]
```diff
@@ -53,8 +53,19 @@ const commonDeclarations = {
   paddingBottom: [ 0 ],
   paddingLeft: [ 0 ],
 
+  minWidth: [ '100%' ],
   maxWidth: [ '100%' ],
-  height: [ 'auto' ],
+  width: [
+    'auto',
+    0,
+    '50%',
```
Comment (asilvas) [inline src/common-declarations.js]: Starting to get into a subjective area where it isn't questionable on being "common". I'd be fine with everything except `50%`.

Comment (lachlanjc) [inline src/common-declarations.js]: Yep, that's definitely subjective, and it'd be easy to start going down the rabbit hole with 25%, 75%, 10%, 20%, 30%, etc. I'll just remove that one. Thanks :)

Comment (lachlanjc) [inline src/common-declarations.js]: Ok, it's out!

**Comment by jxnblk** [comment]
:v:

**Pull request merged**